source/_integrations/influxdb.markdown

```diff
@@ -22,6 +22,8 @@ There is currently support for the following device types within Home Assistant:
 
 - [Sensor](#sensor)
 
+If it seems counters for some entities are not being logged to influxdb, note that by default state objects are logged under their unit, not the entity name. This behavior is controlled via the [`measurement_attr`](https://www.home-assistant.io/integrations/influxdb/#measurement_attr) option.
```

Comment:
I find this a rather weird place to mention this. This is the introduction text where this is added to. A first-time visitor/reader will have no clue what this even means.

Reply:
Any suggestion where to put it? The aim was to increase the visibility of issue, as the page is really long and the forums show people trip-up on this.

Making it a "tip" in configuration section just below might help. Otherwise a new "Troubleshooting" section perhaps?

Actually the entire structure of the page is unintuitive; the word "configure/configuration" is in the name of 7 section titles! It could do with some editorial love.

![image](https://user-images.githubusercontent.com/131580/141859515-fc3ca4df-28a7-4e64-b160-4f0df84d6d96.png)

Reply:
yeah, a trouble shooting section makes sense, or below the configuration section (as the last bit).